README.md

```diff
@@ -1,45 +1,122 @@
-# Website
+# Temporal documentation repository
 
-This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+This repo contains Temporal's open source documentation.
 
-### Prerequisites
+You can view the published documentation at https://docs.temporal.io.
 
-Install [YARN](https://classic.yarnpkg.com/en/docs/install/).
+This repo utilizes the [Docusaurus 2](https://v2.docusaurus.io/) framework, a modern static website generator, which allows you to build and view the documentation locally as you contribute.
 
-### Installation
+## Contribution guidelines
+
+Contributions from the community are highly encouraged.
+
+To contribute please follow these steps.
+
+### Step 1: Fork and clone
+
+To ensure that each change is properly peer reviewed, we request that you follow standard open source contribution processes.
+
+This means that you should start by forking the project into your own GitHub account.
+
+Then, clone your own fork of the repo onto your local machine.
 
 ```
-$ yarn
+$ git clone git@github.com:<your-account>/documentation-legacy.git
 ```
 
-### Local Development
+### Step 2: Add upstream remote
+
+Then add the original repository as a new remote to your local clone.
 
 ```
-$ yarn start
+$ cd documentation-legacy
+$ git remote add upstream git@github.com/temporal/documentation-legacy.git
 ```
 
-This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
+### Step 3: Create a local branch
 
-### Build
+When you are ready to start making changes, sync your local clone with the upstream project and your forked GitHub project.
 
 ```
-$ yarn build
+$ git checkout master
+$ git pull upstream master && git push origin master
+```
+
+Then, create a branch to work on.
```

Comment:
```suggestion
Create a branch to work on.
```

Reply:
I think you could use "then" less especially since it's already structured into steps.